app/views/layouts/_header.html.erb

```diff
@@ -8,11 +8,15 @@
 
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav mr-auto">
-        <li class="nav-item active background-blue">
-          <%= link_to new_user_registration_path, class: 'nav-link nav-icon nav-first' do %>
-            <%= image_tag "BW- CREATE USER ICON.svg", alt:"Create a User" %><p>CREATE USER</p>
+        <% if user_signed_in? %>
```

Comment:
Does this mean that any user can add another user?  I think we only want volunteers and admins to be able to add new users.

Reply:
At first I only checked that `<% if user_can_add_new_member(current_user) %>` But then if there is no currently sign in user present, it got error. So basically first I check sign in user than control user_can_add_new_member before show Create User button